src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java

```diff
@@ -25,7 +47,7 @@ public VoxelSniperConfigLoader(FileConfiguration config) {
     }
 
     /**
-     * Returns if the login message is enabled.
+     * Return if the login message is enabled.
      *
      * @return true if message on login is enabled, false otherwise.
```

Comment:
If javadocs contain (boolean) operational expressions, please highlight them using a code tag, e.g 
`@return {@code true} if message on login is enabled, {@code false} otherwise.`.
The difference is small, but noticeable if you browse Javadocs on the interwebz or in your IDE
Current:
![](https://i.imgur.com/T4xiREe.png)
![](https://i.imgur.com/L63DDHV.png)

With tags:
![](https://i.imgur.com/l3gvz3b.png)
![](https://i.imgur.com/XtDEyPE.png)

Might be just personal preference, but the latter is easier to read.

Reply:
Ty very much for info and details ;)